code/__DEFINES/misc.dm

```diff
@@ -73,8 +73,9 @@
 
 #define BE_CLOSE TRUE		//in the case of a silicon, to select if they need to be next to the atom
 #define NO_DEXTERITY TRUE	//if other mobs (monkeys, aliens, etc) can use this // I had to change 20+ files because some non-dnd-playing fuckchumbis can't spell "dexterity"
-#define NO_TK TRUE
-#define FLOOR_OKAY TRUE     // if you can use it while resting
+#define NO_TK TRUE			// if you can't use it from a distance with telekinesis
+#define FLOOR_OKAY TRUE		// if you can use it while resting
+#define NEED_HANDS TRUE		// if the mob needs an usable hand, regardless of dexterity.
```

Comment:
Expanding on this nonsense is causing me physical pain.

Reply:
I just didn't feel like defying that senseless bandwagon, it's been scratching me an itch too tbh.